.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,12 @@
+// The Dev Container format allows you to configure your environment. At the heart of it
```

Comment:
I hope you don't mind, we use CDEs instead of local machine. It would let anyone to start quickly.